{{cookiecutter.__src_folder_name}}/.github/workflows/test_postgres.yml

```diff
@@ -15,17 +15,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ["ubuntu-20.04"]
+        os: ["ubuntu-latest", "macos-latest", "macos-latest-xlarge", "windows-latest"]
         python_version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-    services:
-      db:
-        image: postgres:14
-        env:
-          POSTGRES_PASSWORD: postgres
-        ports:
-        - 5432:5432
-        # needed because the postgres container does not provide a healthcheck
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        exclude:
+          - os: macos-latest-xlarge
+            python_version: 3.8
+          - os: macos-latest-xlarge
+            python_version: 3.9
+          - os: macos-latest-xlarge
+            python_version: "3.10"
+    steps:
+      - name: Check for MacOS Runner
+      if: matrix.os == 'macos-latest-xlarge'
+      run: brew install postgresql@14
+      - uses: ikalnytskyi/action-setup-postgres@v4
```

Comment:
I think this action may have support for latest now? (i.e is the if needed?)

Reply:
I'll test it and see

Reply:
doesn't seem like it's active yet. I'm getting issues about commands not being available

Reply:
okay, was worth a shot!